verification/verify.go

```diff
@@ -100,23 +105,28 @@ func (sv *SegmentVerifier) Verify(params *Params) (*Params, error) {
 
 	// TODO sig checking; extract from broadcast.go
 
-	if sv.policy.Verifier == nil {
-		return nil, nil
-	}
+	var err error
+	res := &Results{}
+
 	// TODO Use policy sampling rate to determine whether to invoke verifier.
 	//      If not, exit early. Seed sample using source data for repeatability!
-	res, err := sv.policy.Verifier.Verify(params)
+	if sv.policy.Verifier != nil {
+		res, err = sv.policy.Verifier.Verify(params)
+	}
+
+	var pxls []int64
```

Comment:
Doesn't look like this var is used outside of the scope of the if block starting at L123 so maybe we can define it in there which also has the added benefit of avoiding an additional nil check for `res` since the if conditional at L123 already does that check.

Reply:
fixed in 9bd866f73260f75ba6d7e40aa1d5278ee00ffb6a